### ISSUE TITLE
ID 258792: [Programmatic Access -Simon – Value Mapping]: Two <H1> are defined on "Time series graphs /Edit Panel".

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66336]
### **User Experience:**  
Visual impaired users who depend on screen reader will be confused and will be unable to get the title of the page if two `H1` headings are defined within the same page. 
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/ )Sign in with valid credentials. "Home" page appears.  
2. Navigate the Feature Showcase pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item. 
4. Navigate the page "Edit" page which appears. 
5. Navigate to the "All" buttons and select it. 
6. Navigate to "Value Mapping" button and expand it. 
7. Navigate all the controls present under it. 
8. Run AI Tool. Verifybthe issue

**Actual Result:** 
Issue: Two `<H1>` are defined on "Time series graphs /Edit Panel". 
Observation: Screen reader is announcing as heading level 1 "Time series graphs /Edit Panel".

**Expected Result:** 
Only one `H1` Heading tag should be defined on "Time series graphs /Edit Panel".
_Note: This issue exists throughout the application._

### **Severity:**
`SEV2`

### **Area:**
Any Dashboard page

**Comment by evictorero** [comment]
This title `Time series graphs /Edit Panel` does not exist anymore when editing a panel, it is part of the breadcrumb navigation now
![image](https://github.com/grafana/grafana/assets/20256983/ed10ab28-f818-4c46-b947-f8dd4e0eff6a)
